packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -57,16 +180,27 @@ TagModal.propTypes = {
     className: PropTypes.string,
     tableProps: PropTypes.shape({
         [PropTypes.string]: PropTypes.any
-    })
+    }),
+    onSelect: PropTypes.func,
+    onUpdateData: PropTypes.func,
+    pagination: PropTypes.shape({
+        count: PropTypes.number,
+        page: PropTypes.number,
+        perPage: PropTypes.number
+    }),
+    selected: PropTypes.array
 };
 
 TagModal.defaultProps = {
+    loaded: true,
```

Comment:
Not sure, double checking, should this be false?

Reply:
Yeah, that's correct!